lib/renaissance/auctions/auctions.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Renaissance.Auctions do
+  alias Renaissance.{Auction, Repo, Users}
+
+  def create_auction(email, params) do
+    auction =
+      params["auction"]
+      |> add_user_id(email)
+      |> format_end_date()
+      |> format_price()
+
+    Repo.insert(Auction.changeset(%Auction{}, auction))
+  end
+
+  def get_by_title(title) do
+    Repo.get_by(Auction, title: title)
+  end
+
+  defp format_end_date(params) do
+    datetime =
+      "#{params["end_date_day"]}T#{params["end_date_time"]}:00-05:00"
+      |> DateTime.from_iso8601()
+      |> elem(1)
```

Comment:
I think you're using `elem(1)` here to avoid handling the result tuple of either:
`{:ok, date}`, `{:error, reason}`. In which case if you actually want to not handle the error case you can use `DateTime.from_iso8601!()`

Reply:
Resolved with 159a775e493da7136d042a4f76b690b4d8cb2aea